robot/basestation/static/js/rover.js

```diff
@@ -135,36 +135,19 @@ $(document).ready(function () {
     }
   })
 
-  $('#ping-odroid').on('click', function (event) {
-    if (millisSince(lastCmdSent) > PING_THROTTLE_TIME) {
-      appendToConsole('pinging odroid')
-      $.ajax('/ping_rover', {
-        success: function (data) {
-          appendToConsole(data.ping_msg)
-          if (!data.ros_msg.includes('Response')) {
-            appendToConsole('No response from ROS ping_acknowledgment service')
-          } else {
-            appendToConsole(data.ros_msg)
-          }
-        },
-        error: function () {
-          console.log('An error occured')
-        }
-      })
-      lastCmdSent = new Date().getTime()
-    }
+  $('#reboot-button').on('click', function (event) {
+    event.preventDefault()
+    sendRoverCommand('reboot')
   })
-  $('#ping-rover-mcu').on('click', function (event) {
+
+  $('#list-all-cmds').on('click', function(event){
     event.preventDefault()
-    if (millisSince(lastCmdSent) > PING_THROTTLE_TIME) {
-      sendRoverRequest('ping', function (msgs) {})
-      lastCmdSent = new Date().getTime()
-    }
+    printCommandsList()
   })
 
-  $('#reboot-button').on('click', function (event) {
+  $('#stop-all-motors').on('click', function(event){
```

Comment:
When I hit 'O' it sends a rover request to stop all motors, whereas it should be a rover command, and I suppose you want it to be sent on 'Q'. The request vs command thing might be my fault actually but ye.

Reply:
fixed

Reply:
Hitting 'O' still sends rover request

Reply:
Just replacing on line 330 `sendRoverRequest` with `sendRoverCommand` should do the trick?